Website/Resources/Shared/scripts/dnn.dragDrop.js

```diff
@@ -38,7 +38,7 @@
         var $module;
 
         var getModuleId = function ($mod) {
-            var result = $mod.attr("class").match(/DnnModule-([0-9]+)/);
+            var result = $mod.attr("class").match(/DnnModule-([0-9]+)(?:\W|$)/);

```

Comment:
to solve @iJungleboy's edge cases, we can use [`\b`](https://www.regular-expressions.info/wordboundaries.html) to match the word boundary, i.e. `/\bDnnModule-([0-9]+)\b/`

You can see a test at https://jsbin.com/gocokiw/1/edit?js,output

Reply:
@bdukes Thanks. I tested your proposed solution. Learned something new again :-). I committed the change.

Reply:
@bdukes amazing - i've been regexing for forever and never knew about `\b` - very cool! 👍